src/main/java/ca/corefacility/bioinformatics/irida/ria/web/linelist/LineListController.java

```diff
@@ -36,18 +36,20 @@
 import ca.corefacility.bioinformatics.irida.service.sample.MetadataTemplateService;
 import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
 
+import com.google.common.base.Strings;
+
 /**
  * This controller is responsible for AJAX handling for the line list page, which displays sample metadata.
  */
 @Controller
 @RequestMapping("/linelist")
 public class LineListController {
-	private ProjectService projectService;
-	private SampleService sampleService;
-	private MetadataTemplateService metadataTemplateService;
-	private MessageSource messages;
-	private UpdateSamplePermission updateSamplePermission;
-	private ProjectOwnerPermission projectOwnerPermission;
+	private final ProjectService projectService;
```

Comment:
The `final` variable should be removed here as well no?

Reply:
Think I got it all fixed in : c4f7320ebf7b82fad9636119048b65bec9f20718